### PR TITLE
Issue #9568: update example of AST for TokenTypes.SINGLE_LINE_COMMENT

### LINKED ISSUE #9568: update example of AST for TokenTypes.SINGLE_LINE_COMMENT
https://checkstyle.org/apidocs/com/puppycrawl/tools/checkstyle/api/TokenTypes.html#SINGLE_LINE_COMMENT

needs to be extended to show example of AST.
Source to be updated: https://github.com/checkstyle/checkstyle/blob/master/src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java

PLEASE READ for more details: https://github.com/checkstyle/checkstyle/wiki/Good-practice-on-how-to-update-AST-in-TokenTypes

### PR DESCRIPTION
**Pull request by MahmoudAbdelazim**
Closes #9568
image

Code used for Generating example: 
```
mahmoud:~/Desktop/checkstyle$ more Test.java 
// Comment Content
public class A {

}
```
```
mahmoud:~/Desktop/checkstyle$ java -jar checkstyle-8.41-all.jar -T Test.java 
CLASS_DEF -> CLASS_DEF [2:0]
|--MODIFIERS -> MODIFIERS [2:0]
|   |--SINGLE_LINE_COMMENT -> // [1:0]
|   |   `--COMMENT_CONTENT ->  Comment Content\n [1:2]
|   `--LITERAL_PUBLIC -> public [2:0]
|--LITERAL_CLASS -> class [2:7]
|--IDENT -> A [2:13]
`--OBJBLOCK -> OBJBLOCK [2:15]
    |--LCURLY -> { [2:15]
    `--RCURLY -> } [4:0]
```
Expected update for JavaDoc is:
```
SINGLE_LINE_COMMENT -> //
 `--COMMENT_CONTENT ->  Comment Content\n
```

### CODE REVIEW

File: src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java [hunk 1]
```diff
@@ -4022,6 +4022,16 @@
...
      *         +--COMMENT_CONTENT
      * </pre>
+     *
+     * <p>For example:</p>
+     * <pre>
+     * // Comment content
+     * </pre>
+     * <p>parses as:</p>
+     * <pre>
+     * --SINGLE_LINE_COMMENT -&gt; //
```
Comment (romani) [inline src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java:4032]: Remove leading "--" to indent whole code to right

**Review by romani — CHANGES_REQUESTED** [review]
Item:

File: src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java (hunk 1 above)
Comment (MahmoudAbdelazim) [reply src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java:4032]: On it

Comment (MahmoudAbdelazim) [reply src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java]: Done, Please check it again.

**Review by romani — APPROVED** [review]

**Pull request merged**